lib/base_service.js

```diff
@@ -31,6 +31,15 @@ function BaseService(options) {
     this.serviceReturns = null;
 }
 
+BaseService.prototype.cleanup = function() {
```

Comment:
Could we call this method also `stop` and then override it in Master and Worker and call a superclass method with something like `BaseService.prototype.cleanup.call(this)`? It's gonna be cleaner in OOP sense and also when we switch to class syntax it should be nicer. Not a strong opinion though